app/src/main/java/de/grobox/transportr/ui/TimeDateFragment.kt

```diff
@@ -38,19 +39,22 @@ import java.util.Calendar.*
 class TimeDateFragment : DialogFragment(), OnDateSetListener, OnTimeChangedListener {
 
     private var listener: TimeDateListener? = null
+    private var departure: Boolean? = null
```

Comment:
If this is always set in `onCreate()` it could be `lateinit` and `Boolean` (non-null).

Reply:
Ah it being `null` means don't show departure stuff. Maybe add a quick documentation for that?

Reply:
> Ah it being `null` means don't show departure stuff. Maybe add a quick documentation for that?

Exactly, a little comment would be good here.

Reply:
Added in updated commit.